test/fuzz/BurnTest.t.sol

```diff
@@ -579,27 +579,31 @@ contract BurnTest is Fixture, FunctionUtils {
         if (stableAmount == 0) return;
 
         // _logIssuedCollateral();
-        uint256 amountOut = transmuter.quoteIn(stableAmount, address(agToken), _collaterals[fromToken]);
-        // This will crash if the
-        if (amountOut != 0) {
-            uint256 reflexiveAmountStable = transmuter.quoteOut(amountOut, address(agToken), _collaterals[fromToken]);
-            uint256 reflexiveAmountOut = transmuter.quoteIn(
-                reflexiveAmountStable,
-                address(agToken),
-                _collaterals[fromToken]
-            );
-
-            if (amountOut > _minWallet / 10 ** (18 - IERC20Metadata(_collaterals[fromToken]).decimals())) {
-                _assertApproxEqRelDecimalWithTolerance(
-                    reflexiveAmountOut,
+        try transmuter.quoteIn(stableAmount, address(agToken), _collaterals[fromToken]) returns (uint256 amountOut) {
```

Comment:
you can instead do the `checkAmounts` in here instead of random revert

Reply:
Yes but I have no way to know the amountOut without exactly replicating the code